vaccine_feed_ingest_schema/location.py

```diff
@@ -35,8 +166,8 @@ class LatLng(BaseModel):
     },
     """
 
-    latitude: float
-    longitude: float
+    latitude: float = Field(gt=-90.0, lt=90.0)
+    longitude: float = Field(gt=-180.0, lt=180.0)
```

Comment:
It looks like no in-scope land falls on the 180th meridian (or at the poles), so we're good!

Reply:
I'll changes these to `ge` and `le` just to be safe